src/zcc/zcc.c

```diff
@@ -2733,34 +2740,10 @@ int find_zcc_config_fileFile(char *arg, int gc, char *buf, size_t buflen)
 		* when
 		*/
 		return (gc);
-	}
-	/* Okay, nowt specified so get the old style entry */
-	cfgfile = getenv("ZCCFILE");
-	if (cfgfile != NULL) {
-		if (strlen(cfgfile) > FILENAME_MAX) {
-			fprintf(stderr, "Possibly corrupt env variable ZCCFILE\n");
-			exit(1);
-		}
-		snprintf(buf, buflen, "%s", cfgfile);
-		return (gc);
-	}
-	/*
-	* Last resort!
-	* New style config take ZCCCFG/zcc.cfg
-	*/
-	cfgfile = getenv("ZCCCFG");
-	if (cfgfile != NULL) {
-		if (strlen(cfgfile) > FILENAME_MAX) {
-			fprintf(stderr, "Possibly corrupt env variable ZCCCFG\n");
-			exit(1);
-		}
-		snprintf(buf, buflen, "%s/zcc.cfg", cfgfile);
-	}
-	else {
-		// Fall back to a sensible default
-		snprintf(buf, buflen, "%slib/config/zcc.cfg", c_install_dir);
-	}
-	return (gc);
+	} 
+	// Without a config file, we should just exit
```

Comment:
Did we want to print some error message as well, so the user knows what is wrong?